notmuch-addrlookup.c

```diff
@@ -354,6 +386,19 @@ main (int argc, char **argv)
       return EXIT_FAILURE;
     }
   g_option_context_free (option_context);
+  const struct PrintFormat *output_format = NULL;
+  for (unsigned i = 0; i < G_N_ELEMENTS(s_formats); i++) {
+      if (strcmp(s_formats[i].name, output_format_name) == 0) {
+          output_format = &s_formats[i];
+          break;
+      }
+  }
+
+  if (!output_format)
+    {
+      g_printerr ("%s: %s is an %s.\n", g_get_prgname (),output_format_name, "invalid format name");
```

Comment:
You could put the `"invalid format name"` part in the format string directly:

```suggestion
      g_printerr ("%s: %s is an invalid format name.\n", g_get_prgname (), output_format_name);
```

Reply:
ok, so that was just plain not thinking :D